_im-plugin/performance.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+title: Performance considerations
+nav_order: 41
+has_children: false
+---
+
+# Performance considerations while tuning Opensearch for indexing speed
```

Comment:
```suggestion
# Tuning your cluster for indexing speed
```

Reply:
accepted your feedback, and updated in the new location in the doc.